package.json

```diff
@@ -43,12 +43,11 @@
         "url": "https://github.com/PluginPal/strapi-plugin-publisher/issues"
     },
     "dependencies": {
-        "@strapi/design-system": "2.0.0-rc.11",
-        "@strapi/icons": "2.0.0-rc.11",
-        "@strapi/utils": "5.2.0",
-        "@strapi/strapi": "^5.2.0",
+        "@strapi/design-system": "^2.0.0-rc.11",
+        "@strapi/icons": "^2.0.0-rc.11",
```

Comment:
If we have these in the peerDependencies, we can remove them from the dependencies.

Sorry to be nitpicking about this, but it is a rather delicate component of the plugin which can easily be done wrong.